src/components/TrackTime/helpers.ts

```diff
@@ -35,3 +44,38 @@ export function customizedTime(time: FormattedTime, locale: Locale): string {
 
   return `${formattedHours}${hours} ${formattedMinutes}${minutes} ${formattedSeconds}${seconds}`
 }
+
+export function customizedPeriod(
+  dates: TimePeriod[],
+  language: Language,
+): string {
+  if (!dates.length) return ''
+
+  moment.locale(language)
+
+  const { startDate, endDate } = dates.at(-1)
+
+  const formattedStartDate = moment(startDate).format(DATE_DAY_FORMAT)
+  const formattedStartTime = moment(startDate).format(DATE_TIME_FORMAT)
+  const formattedEndDate = moment(endDate).format(DATE_DAY_FORMAT)
+  const formattedEndTime = moment(endDate).format(DATE_TIME_FORMAT)
+
+  if (formattedStartDate === formattedEndDate) {
+    return `${formattedStartDate}, ${formattedStartTime} - ${formattedEndTime}`
+  }
+
+  return `${formattedStartDate} ${formattedStartTime} - ${formattedEndDate} ${formattedEndTime}`
+}
+
+export function totalElapsedTime(dates: TimePeriod[], locale: Locale): string {
+  const now = Date.now()
+
+  const totalTime = getTotalTimeForDate(now, dates)
+
+  const customTime = customizedTime(formatTime(totalTime, false), locale)
+
+  return customTime
+    .split(' ')
+    .map(item => (item.startsWith('0') ? '' : item))
```

Comment:
The `split `and `map `function removes any parts of the string that start with '0', but it seems a bit fragile (e.g., "08:30" might get transformed unexpectedly). Check it.

Reply:
const customTime = customizedTime(formatTime(totalTime, false), locale)

I pass to the function ormatTime(totalTime, false)
and with false

this function returns in the format 0h 0m 4s